docs/tutorials/migrate-from-mbed-os-2-to-5.md

```diff
@@ -48,6 +48,20 @@ To determine the success of migration, run:
 
 `mbed compile -m [platform] -t [toolchain]`
 
+The general outline for updating to Mbed OS 5 with the [Mbed Online Compiler](https://os.mbed.com/docs/latest/tools/arm-mbed-online-compiler.html) is:
+
+1. Open your project in the online compiler
+1. Right click on `mbed` and select "Delete..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-delete-mbed.png)</span>
+1. If your project includes `mbed-rtos`, you will also need to delete this library to successfully update to Mbed OS 5. Right click on `mbed-rtos` and select "Delete..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-delete-mbed-rtos.png)</span>
+1. Right click on the name of your project and hover over "Import Library" then select "From URL ..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-import-url.png)</span>
+1. Copy and paste the URL for Mbed OS 5 `https://github.com/armmbed/mbed-os` then click "Import"
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/import-mbed-os.png)</span>
+
+To determine the success of migration, select your board in the top right corner of the Online Compiler, then click  **Compile**.
+
```

Comment:
Can't comment on line #142 below, but we don't use '/', so I'm assuming that you mean "replace the SDA [and] SCL pins with the ones present on the K64" and that you mean "D14 [and] D15 work"??  Let me know if the '/' means 'and' in these two places.

Reply:
Also, in line #290 below, after "Line 114 of the `AD7790.h` header file now looks like this:" I think there's an extra space after the first '(' below in line #290 - the others don't have this - is this an error?

Reply:
@jen3andruska Sorry, I'm confused about your comments. Which line numbers are you referring to? Those links go to PR's

Reply:
@yennster Ok, its now line #300 - it says: 
AD7790( float reference_voltage, PinName CS, PinName MOSI, PinName MISO, PinName SCK);
Is there supposed to be a space after the '('   after AD7790?
The other header files do not have a space after the '(' and thought it should probably be deleted

Reply:
@jen3andruska Oh yeah, that space is a typo. Shouldn't cause any compiling issues, however

Reply:
@jen3andruska Fixed